deploy/helm/sifnode/templates/sifnode-pvc.yml

```diff
@@ -0,0 +1,21 @@
+{{- if and .Values.persistence.enabled (not .Values.persistence.existingClaim) (not .Values.persistence.hostPath) }}
+kind: PersistentVolumeClaim
+apiVersion: v1
+metadata:
+  name: {{ template "sifnode.fullname" . }}
+  labels:
+    {{- include "sifnode.labels" . | nindent 4 }}
+spec:
+  accessModes:
+    - {{ .Values.persistence.accessMode | quote }}
+  resources:
+    requests:
+      storage: {{ .Values.persistence.size | quote }}
+  {{- if .Values.persistence.storageClass }}
+  {{- if (eq "-" .Values.persistence.storageClass) }}
+storageClassName: ""
+  {{- else }}
+storageClassName: "{{ .Values.persistence.storageClass }}"
+  {{- end }}
+  {{- end }}
+  {{- end -}}
```

Comment:
@intl-man can we maybe just intend correctly the if statements, or maybe that is how GitHub displays the code?

Reply:
@ilchebedelovski Done.